cypress/integration/examples/basic.spec.ts

```diff
@@ -72,7 +72,6 @@ context('Basic', () => {
     cy.get('body')
       .type('{UpArrow}')
       .url()
-      .should('eq', 'http://localhost:3030/6')
```

Comment:
Oops, I made a mistake and accidentally deleted this line.

Reply:
You can do another PR to bring it back. Thanks